pkg/ocm/clusters.go

```diff
@@ -558,3 +558,62 @@ func (c *Client) createClusterSpec(config Spec, awsClient aws.Client) (*cmv1.Clu
 
 	return clusterSpec, nil
 }
+
+func (c *Client) HibernateCluster(clusterKey string, creatorARN string) error {
+	reporter := rprtr.CreateReporterOrExit()
+
+	err := c.IsHibernateCapabilityEnabled()
+	if err != nil {
+		return err
+	}
+
+	// Get the cluster to check the state
+	cluster, err := c.GetCluster(clusterKey, creatorARN)
+	if err != nil {
+		reporter.Errorf("Failed to get cluster '%s': %v", clusterKey, err)
+		os.Exit(1)
```

Comment:
This should be done in the command, otherwise third-party libraries importing this (e.g. osde2e) will crash their entire pipelines if it gets here. Here you can just `return fmt.Errorf("Failed to get cluster '%s': %v", clusterKey, err)` and use the reporter/os.Exit in the command.